grails-app/services/org/broadinstitute/orsp/PersistenceService.groovy

```diff
@@ -53,4 +53,10 @@ class PersistenceService {
         consentCollectionLink?.delete(flush: true)
     }
 
+    void updateCollectionLinkStatus(String consentKey, String projectKey, String status) {
+        ConsentCollectionLink.executeUpdate("update ConsentCollectionLink ccl set ccl.status = :status where ccl.projectKey = :projectKey and ccl.consentKey = :consentKey",
```

Comment:
nit: This is HQL, not sql, and will work fine. I'm not requesting that you change this, but wanted to point out that this is a different pattern than what we've been striving for, i.e. moving toward using sql directly in persistence services. In general, anything that is GORM or hql has not been nearly as performant as raw sql. It's trivial to write these updates that way, but when we use hql/gorm, it goes through a significant layer of ORM manipulation overhead. For simple updates like this, it's likely that you won't see the impact. In larger updates/queries, you will definitely see a performance impact.

Reply:
:+1: I moved to sql